libraries/AP_InertialSensor/AP_InertialSensor_Invensensev3.cpp

```diff
@@ -98,13 +106,41 @@ static const float GYRO_SCALE = (0.0174532f / 16.4f);
 #define INV3REG_MREG1_FIFO_CONFIG5 0x1
 #define INV3REG_MREG1_SENSOR_CONFIG3 0x06
 
+#define INV3REG_456_WHOAMI          0x72
```

Comment:
I feel like the changes are significant enough that perhaps this needs its own backend with some kind of common base class (ditto the 42760)